app/pods/publications/to-treat/template.hbs

```diff
@@ -1,89 +1,109 @@
+                </td>
+              {{else}}
+                <td>
+                  <p>
+                    {{#if row.shortTitle}}
+                      {{row.shortTitle}}
+                    {{else}}
+                      {{row.title}}
+                    {{/if}}
+                  </p>
+                </td>
+                <td>
+                  {{#if row.loadDocumentsCount.isRunning}}
+                    {{utils/simple-spinner}}
                   {{else}}
-                    {{row.title}}
+                    {{row.documentsCount}}
                   {{/if}}
-                </p>
-              </td>
-              <td>
-                {{#if row.loadDocumentsCount.isRunning}}
-                  {{utils/simple-spinner}}
-                {{else}}
-                  {{row.documentsCount}}
-                {{/if}}
-              </td>
-              <td class="vl-u-align-right">
-                {{#if (await row.publicationFlow)}}
-                  <LinkTo
-                          class=""
-                          @route="publications.publication"
-                          @model={{ await row.publicationFlow.id }}
-                  >
-                    {{t "publications.to" }}
-                  </LinkTo>
-                {{else}}
-                  <button class="vl-button vl-button--tertiary vl-button--icon-before" type="button"
-                    {{on "click" (fn this.startPublication row) }}
-                  >
-                    {{t "start-publication"}}
-                  </button>
-                {{/if}}
-              </td>
-            {{/if}}
-          </c.body>
-        </table.content>
-      </DataTable>
-    {{else}}
-      {{t "no-results-found"}}
+                </td>
+                <td class="">
+                  <div class="auk-button-toolbar">
+                    {{#if (await row.publicationFlow)}}
+                      <LinkTo
+                              class=""
+                              @route="publications.publication"
+                              @model={{ await row.publicationFlow.id }}
+                      >
+                        {{t "publications.to" }}
+                      </LinkTo>
+                    {{else}}
+                      <WebComponents::AuButton
+                                               @skin="tertiary"
+                                               @layout="icon-only"
+                                               data-test-start-publication
+                                              {{on "click" (fn this.startPublication row)}}>
+                        {{t "start-publication"}}
+                      </WebComponents::AuButton>
+                    {{/if}}
+                  </div>
+                </td>
+              {{/if}}
+            </c.body>
+          </table.content>
+        </DataTable>
+        <div class="vl-u-spacer--small"></div>
```

Comment:
Best geen `vl` css meer gebruiken.. kijken of we een spacer--small hebben in de AUK utillity css